[ONNX] minor clarifications of docstrings (#69260)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #69550
* __->__ #69549
* #69548
* #69547
* #69546
* #69544
* #68491
* #68490

[ONNX] minor clarifications of docstrings

1. Make description of ONNX_ATEN_FALLBACK more accurate (after #67460).
2. Specify minimum and maximum values for opset_version. This is pretty
   important information and we should make users dig through source
   code to find it.

Differential Revision: [D32994267](https://our.internmc.facebook.com/intern/diff/D32994267)

BowenBao committed:
[ONNX] minor clarifications of docstrings (#69260)

[ONNX] minor clarifications of docstrings

1. Make description of ONNX_ATEN_FALLBACK more accurate (after #67460).
2. Specify minimum and maximum values for opset_version. This is pretty
   important information and we should make users dig through source
   code to find it.

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994267).

BowenBao committed:
Update on "[ONNX] minor clarifications of docstrings (#69260)"


[ONNX] minor clarifications of docstrings

1. Make description of ONNX_ATEN_FALLBACK more accurate (after #67460).
2. Specify minimum and maximum values for opset_version. This is pretty
   important information and we should make users dig through source
   code to find it.

Differential Revision: [D32994267](https://our.internmc.facebook.com/intern/diff/D32994267)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994267).

BowenBao committed:
Update on "[ONNX] minor clarifications of docstrings (#69260)"


[ONNX] minor clarifications of docstrings

1. Make description of ONNX_ATEN_FALLBACK more accurate (after #67460).
2. Specify minimum and maximum values for opset_version. This is pretty
   important information and we should make users dig through source
   code to find it.

Differential Revision: [D32994267](https://our.internmc.facebook.com/intern/diff/D32994267)

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32994267).
